readme.md

```diff
@@ -238,7 +259,7 @@ const limiter = rateLimit({
 
 The response body to send back when a client is rate limited.
 
-May be a `string`, JSON object, or any other value that Express's
+May be a `string`, JSON object, function, or any other value that Express's
 [response.send](https://expressjs.com/en/4x/api.html#response.send) method
 supports.
 
```

Comment:
Below the `Defaults to ...`, it would be helpful to add an example of using the function, similar to the `max` option above:

---

An example of using a function:

````ts
const isPremium = async (user) => {
	// ...
}

const limiter = rateLimit({
	// ...
	message: async (request, response) => {
		if (await isPremium(request.user)) return 'You can only make 10 requests every hour.'
		else return 'You can only make 5 requests every hour.'
	},
})
```